src/modules/data/blocks/ticket/sagas.js

```diff
@@ -204,20 +204,22 @@ export function* setTicketInitialState( action ) {
 	if ( ticketId !== 0 ) {
 		yield all( [
 			put( actions.setTicketId( clientId, ticketId ) ),
-			put( actions.fetchTicket( clientId, ticketId ) ),
+			call( fetchTicket, { payload: { clientId, ticketId } } ),
```

Comment:
I do not think this will proceed with that new `while` loop in the `fetchTicket` saga, is that your intention?

Reply:
The `while` loop is not in the `fetchTicket` saga, it's in the `saveTicketWithPostSave` saga, which is `fork`ed after this. I changed this to a `call` because the `put` would just dispatch the action and not wait for the `fetch` to finish. Then the `handleTicketDurationError` below runs before the `fetch` isn't even finished, which is not what was intended. That's not the important part though, I actually need the `saveTicketWithPostSave` just below that to run after the `fetch`.

Reply:
What's happening here now is all tickets are saving on post save. Before, you had set it up so it save only if it was synced. But because we need to save the order of the tickets, we need to save the tickets on each post save.

Reply:
Oh I see, sorry about that. I thought I expanded the diff all the way - carry on :)